File: app/src/main/java/com/lms/mpasho_lms_news/models/News.java
package com.lms.mpasho_lms_news.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class News {

    @SerializedName("articles")
    @Expose
    private List<Article> article;

    public List<Article> getArticle() {
        return article;
    }

}

File: app/src/main/java/com/lms/mpasho_lms_news/models/News.kt
package com.lms.mpasho_lms_news.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class News {

    @SerializedName("status")
    @Expose
    var status: String? = null

    @SerializedName("totalResult")
    @Expose
    var totalResult: Int = 0

    @SerializedName("articles")
    @Expose
    var article: List<Article>? = null
}